### ISSUE TITLE
Add Travis-CI Build Status Badge To readme.md

### ISSUE THREAD

**Issue by kb1lqc** [FaradayRF/faradayio#29]
Add a build status badge to the `readme.md` file to reflect the build status on Travis-CI. This will indicate the latest build status of the `master` branch of `faradayio`.

[![Build Status](https://travis-ci.org/FaradayRF/faradayio.svg?branch=master)](https://travis-ci.org/FaradayRF/faradayio)

More info can be found here: https://docs.travis-ci.com/user/status-images/

## Markdown to Add
The following markdown needs to be added to the `readme.md` file:

```
[![Build Status](https://travis-ci.org/FaradayRF/faradayio.svg?branch=master)](https://travis-ci.org/FaradayRF/faradayio)
```

**Comment by kb1lqc** [comment]
Don't worry about the failing status at this time. The `develop` branch on my personal fork is about to be pushed and is the initial code volley for this project. Future PR's will not nearly be as drastic and huge.

**Comment by kb1lqc** [comment]
This is a real quick one for anyone interested @lqdev!

**Comment by lqdev** [comment]
Okay. Taking a look at the Travis link.

**Comment by kb1lqc** [comment]
Closing ticket as this was merged with `master` in #31.
